larsimrad/BxDecay0/Decay0Gen_module.cc

```diff
@@ -165,37 +167,47 @@ namespace evgen{
               p.print(std::cout);
               throw cet::exception("Decay0Gen") << "Particle above is weird, cannot recognise it.";
             }
+            
             if ((p.is_positron() or p.is_electron()) and t_electron == 0) {
               t_electron = p.get_time();
             }
             if (p.is_alpha() and t_alpha == 0) {
               t_alpha = p.get_time();
             }
             track_id--;
-
             TLorentzVector mom(p.get_px()/1000.,
                                p.get_py()/1000.,
                                p.get_pz()/1000.,
                                sqrt(p.get_p()*p.get_p() + mass*mass)/1000.);
-
+            
```

Comment:
Looks like some trailing whitespace has been added a few places--see if you can remove these (usually `git diff --ws-error-highlight=all ...` is sufficient).

Reply:
I ran a `removeWhiteSpace.sh` on this file and commited the result.